### DISCUSSION TITLE
saf-training/courses/profile-dev-test/06

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#75] [Ideas]
# saf-training/courses/profile-dev-test/06

Docker Suite Setup 1. Make sure Docker or Podman is running 2. Login to your docker registry 3. Clone the repository 4. Go into the profile repository root dir 5. bundle install...

https://mitre.github.io/saf-training/courses/profile-dev-test/06.html

**Comment by seanlongcc** [comment]
We learn that there is a `vanilla` and `hardened` command, but only go through and use the `vanilla` command. An explanation or usage example of the `hardened` command would clear up some confusion of what its for.
